### PR TITLE
Make QR install links better

### PR DESCRIPTION
**Pull request by cobbal**
The big thing here is unifying the ad-hoc builds.

The obvious way to do that would be to run archive 3 times, but that is really slow. Xcode will always do a clean build for archiving, so it has to compile Core 3x.

Instead, there's a new archive target "All" which has dependencies on all 3 apps. When the archive is created it's a "generic" archive meaning that the apps can't be exported from it. To fix it, a shell script duplicates the archive bundle 3 times, and then tweaks them to make the bundle app-specific.

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Core/Core/PageViewAnalytics/PageViewEventController.swift:42:51: Unneeded NotificationCenter Removal Violation: Observers are automatically unregistered on dealloc (iOS 9 / macOS 10.11) so you should't call `removeObserver(self)` in the deinit. (unneeded_notification_center_removal)
    /Users/vagrant/git/Core/Core/Extensions/UITableViewCellExtensions.swift:25:9: warning: Computed Accessors Order Violation: Computed properties should declare first the getter and then the setter. (computed_accessors_order)
    /Users/vagrant/git/Core/Core/UIViews/TokenLabelView.swift:65:9: warning: Computed Accessors Order Violation: Computed properties should declare first the getter and then the setter. (computed_accessors_order)
    ⚠️  /Users/vagrant/git/Core/Core/UIViews/KeyboardTransitioning.swift:39:51: Unneeded NotificationCenter Removal Violation: Observers are automatically unregistered on dealloc (iOS 9 / macOS 10.11) so you should't call `removeObserver(self)` in the deinit. (unneeded_notification_center_removal)
    /Users/vagrant/git/Core/Core/Profile/Settings/NotificationCategoriesViewController.swift:77:51: warning: Unneeded NotificationCenter Removal Violation: Observers are automatically unregistered on dealloc (iOS 9 / macOS 10.11) so you should't call `removeObserver(self)` in the deinit. (unneeded_notification_center_removal)
    /Users/vagrant/git/Core/Core/Files/FilePicker/FilePickerProgressView.swift:38:9: warning: Computed Accessors Order Violation: Computed properties should declare first the getter and then the setter. (computed_accessors_order)
    ⚠️  /Users/vagrant/git/Core/Core/AppEnvironment/SessionDefaults.swift:54:9: Computed Accessors Order Violation: Computed properties should declare first the getter and then the setter. (computed_accessors_order)
    /Users/vagrant/git/Core/Core/Modules/ModuleItem.swift:64:28: warning: Empty Enum Arguments Violation: Arguments can be omitted when matching enums with associated types if they are not used. (empty_enum_arguments)
    [student]
    ⚠️  /Users/vagrant/git/Student/Student/Submissions/SubmissionDetails/SubmissionDetailsViewController.swift:48:51: Unneeded NotificationCenter Removal Violation: Observers are automatically unregistered on dealloc (iOS 9 / macOS 10.11) so you should't call `removeObserver(self)` in the deinit. (unneeded_notification_center_removal)
    [teacher]
    /Users/vagrant/git/rn/Teacher/ios/Teacher/Attendance/AttendanceViewController.swift:252:79: warning: Empty Enum Arguments Violation: Arguments can be omitted when matching enums with associated types if they are not used. (empty_enum_arguments)
</details>
<details><summary>:x: XCTest failed: CoreTests/RichContentEditorViewControllerTests/testMediaRetry</summary>


    Assertion Failure: RichContentEditorViewControllerTests.swift:56: Asynchronous wait failed: Exceeded timeout of 10 seconds, with unfulfilled expectations: "updated".
</details>


[MBL-14542](https://instructure.atlassian.net/browse/MBL-14542)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Teacher.ipa</summary>
https://app.bitrise.io/artifact/48861339/p/451f40f0db90e52aa4c88ad835a6bc00
</details><details><summary>Parent.ipa</summary>
https://app.bitrise.io/artifact/48861230/p/6c545308ccabd8d3c5abc4d81ea572ab
</details><details><summary>Student.ipa</summary>
https://app.bitrise.io/artifact/48861275/p/d78b5b1c3acc6bbf218a92961aac4401
</details>

File: scripts/swift/BuildLinks.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          }
+
+        var body = """
+          <!-- \(magicString) -->
+          """
+        for app in apps {
+            body += """
+                <details><summary>\(app.file)</summary>
+                \(app.url)
+                </details>
```
Comment (vanwagonet) [inline scripts/swift/BuildLinks.swift:60]: The details, summary pattern means I need 2 clicks. Can we just make it a link?
`"<a href=\"\(app.url)\">\(app.file)</a>"`

Comment (cobbal) [reply scripts/swift/BuildLinks.swift:60]: That was temporary testing code. It will be replaced by a QR image shortly

**Review by vanwagonet — APPROVED** [review]
Works well. QA+1

**Pull request merged**